swagger.yaml

```diff
@@ -4290,13 +7455,78 @@ definitions:
         description: JSON-formatted string containing the usage information by feature name
         x-go-custom-tag: gorm:"type:text"
 
+  infraenv-create-params:
+    type: object
+    properties:
+      name:
+        type: string
+        description: Name of the InfraEnv.
+      proxy:
+        $ref: "#/definitions/proxy"
+      additional_ntp_sources:
+        type: string
+        description: A comma-separated list of NTP sources (name or IP) going to be added to all the agents.
+      ssh_authorized_key:
+        type: string
+        description: SSH public key for debugging the installation.
+      pull_secret:
+        type: string
+        description: The pull secret obtained from Red Hat OpenShift Cluster Manager at cloud.redhat.com/openshift/install/pull-secret.
+      static_network_config:
+        type: array
+        items:
+          $ref: '#/definitions/agent_static_network_config'
+      image_type:
+        $ref: '#/definitions/image_type'
+      ignition_config_override:
+        type: string
+        description: Json formatted string containing the user overrides for the initial ignition config
+      cluster_id:
+        type: string
+        format: uuid
+        description: If set, all agents that register will be associated with the specified cluster
```

Comment:
Can the user move the agent to a different cluster as well? 
If cluster is already installed will it install a day2 host? 
On patch i expect hosts that are not installed to move to a different cluster? until which statge?
Maybe need to document those flows somewhere

Reply:
I think it's covered (if not should be) at the late binding enhancement,
you can change cluster_id of agent to some cluster and then it will be validated by the cluster type you trying to bind it to, if it's to a day 1 cluster, it's only if the installation didn't start yet,
If you try to bind it to a day 2 cluster then it's like adding a new host today.